doc/playbook.rst

```diff
@@ -79,66 +87,71 @@ From the ``main`` branch you can release a new major version (``X.0.0``), a new
-* Science Pipelines version and rubin-env version
-* Supported `APDB schema`_ and `ApdbSql`_/`ApdbCassandra`_/`ApdbCassandraReplica`_ versions (see `DMTN-269`_ for rationale).
-  A stack quoting a given minor version is compatible with *older* APDBs of that major version but not necessarily newer ones; for example, a release whose baseline is APDB schema 1.4.0 can access a schema 1.0.0 or 1.4.1 database, but not schema 1.5.
-* Supported `Butler dimensions-config`_ versions
-* The `Alerts schema`_ version used for output (see `DMTN-093`_ for details)
+   Select **Generate Release Notes**.
+   This will generate a list of commit summaries and of submitters.
+   Add text as follows:
 
-.. _DMTN-093: https://dmtn-093.lsst.io/#alertmanagement
+   * Any specific motivation for the release (for example, including a specific feature, preparing for a specific observing run)
+   * Science Pipelines version and rubin-env version
+   * Supported `APDB schema`_ and `ApdbSql`_/`ApdbCassandra`_ versions (see `DMTN-269`_ for rationale).
+     You do *not* need to consider the `ApdbCassandraReplica` version.
+     A stack quoting a given minor version is compatible with *older* APDBs of that major version but not necessarily newer ones; for example, a release whose baseline is APDB schema 1.4.0 can access a schema 1.0.0 or 1.4.1 database, but not schema 1.5.
+   * Supported `Butler dimensions-config`_ versions
+   * The `Alerts schema`_ version used for output (see `DMTN-093`_ for details)
 
-.. _DMTN-269: https://dmtn-269.lsst.io/
+   .. _DMTN-093: https://dmtn-093.lsst.io/#alertmanagement
 
-.. _Butler dimensions-config: https://pipelines.lsst.io/v/daily/modules/lsst.daf.butler/dimensions.html#dimension-universe-change-history
+   .. _DMTN-269: https://dmtn-269.lsst.io/
 
-.. _APDB schema: https://github.com/lsst/sdm_schemas/blob/main/python/lsst/sdm/schemas/apdb.yaml#L4
+   .. _Butler dimensions-config: https://pipelines.lsst.io/v/daily/modules/lsst.daf.butler/dimensions.html#dimension-universe-change-history
 
-.. _ApdbSql: https://github.com/lsst/dax_apdb/blob/main/python/lsst/dax/apdb/sql/apdbSql.py#L71-L75
+   .. _APDB schema: https://github.com/lsst/sdm_schemas/blob/main/python/lsst/sdm_schemas/schemas/apdb.yaml#L4
 
-.. _ApdbCassandra: https://github.com/lsst/dax_apdb/blob/main/python/lsst/dax/apdb/cassandra/apdbCassandra.py#L84-L88
+   .. _ApdbSql: https://github.com/lsst/dax_apdb/blob/main/python/lsst/dax/apdb/sql/apdbSql.py#L72-L76
 
-.. _ApdbCassandraReplica: https://github.com/lsst/dax_apdb/blob/main/python/lsst/dax/apdb/cassandra/apdbCassandraReplica.py#L52-L56
+   .. _ApdbCassandra: https://github.com/lsst/dax_apdb/blob/main/python/lsst/dax/apdb/cassandra/apdbCassandra.py#L85-L89
 
-.. _Alerts schema: https://github.com/lsst/alert_packet/blob/main/python/lsst/alert/packet/schema/latest.txt
+   .. _Alerts schema: https://github.com/lsst/alert_packet/blob/main/python/lsst/alert/packet/schema/latest.txt
```

Comment:
Please double-check these links, I think you overwrote the fixes from cbb1233be5fed1366eeb3916fe55a254cb34f8eb when rebasing.